### PR TITLE
refactor/service-agreements

### PR DESCRIPTION
**Pull request by diminator**
## Description

Break all the things

## Is this PR related with an open issue?

Related to Issue #

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [x] Breaking change (fix or feature that would cause existing functionality to change)
 
## Checklist:

- [x] ~Follows~Defines the code style of this project.
- [x] Tests Cover Changes
- [ ] Documentation

#### Funny gif

funny gif

### CODE REVIEW

File: contracts/SLA/ServiceAgreement.sol [hunk 1]
```diff
@@ -7,7 +7,7 @@ import 'openzeppelin-solidity/contracts/cryptography/ECDSA.sol';
 @author Team: Ahmed Ali, Samer Sallam
 */
 
-contract ServiceAgreement {
+contract ServiceExecutionAgreement {
```
Comment (H34D) [inline contracts/SLA/ServiceAgreement.sol]: changing this is breaking, therefor this is not a refactoring anymore and will lead to version `0.6.0`

**Comment by H34D** [comment]
touches #208

File: contracts/OceanToken.sol [hunk 2]
```diff
@@ -14,6 +14,32 @@ contract OceanToken is ERC20Capped, ERC20Detailed {
 
     using SafeMath for uint256;
 
+<<<<<<< HEAD
```
Comment (aabdulwahed) [inline contracts/OceanToken.sol]: @aj07 can you please solve this conflict

Comment (aj07) [reply contracts/OceanToken.sol]: @aabdulwahed updated. Thanks for pointing out

Comment (H34D) [reply contracts/OceanToken.sol]: solved

**Pull request merged**